Sources/App/Models/Entities/AppUser/Extensions/AppUser+JWTKeychain.swift

```diff
@@ -4,8 +4,12 @@ import Reset
 import Sugar
 import Vapor
 
-extension AppUser: JWTCustomPayloadKeychainUserType {
+// TODO: split up these conformances into separate ones, each with it's own file, eg. Updatable, Loginable, Creatable, etc. Remember to check for unnecessary imports when splitting stuff into separate files
```

Comment:
Please try to apply the things from this TODO comment and remove it afterwards :)